hazelcast/src/main/java/com/hazelcast/jet/accumulator/PickAnyAccumulator.java

```diff
@@ -47,11 +46,9 @@ public PickAnyAccumulator() {
      * the parameters.
      *
      * @param value the picked object
-     * @param count the count of accumulated objects
      */
-    public PickAnyAccumulator(T value, long count) {
+    public PickAnyAccumulator(T value) {
```

Comment:
Are the changes in this file affect the public API we offer to the users?

Reply:
OK, I see. It is still a breaking change but a less risky one, one we have to do to fix a defect. I think it should be acceptable to have it then :+1:

Reply:
Exactly, it's a bugfix. The class is Core API, that is public API. But it's supposed to be used through `AggregateOperations.pickAny()`. From there, the `deduct` method documented as being optional. We had to remove that method, because it was implemented incorrectly and cannot be implemented correctly.